### PR TITLE
Make helm chart imagePullPolicy configurable

### PR DESCRIPTION
**Pull request by kerthcet**
Signed-off-by: kerthcet <kerthcet@gmail.com>

/kind feature

We want to make volcano helm chart imagePullPolicy configurable, in case we want to make sure that we're always 
pulling the latest images.

### CODE REVIEW

**Comment by hwdef** [comment]
I think we still need this, please reopen
@kerthcet @william-wang @Thor-wl

**Comment by kerthcet** [comment]
Rebased the PR in case of you agree to merge this one.

**Comment by hwdef** [comment]
Please make the CI happy and rebase your code, I will add lgtm tag for this pr later.

**Review by Thor-wl — APPROVED** [review]

**Review by Thor-wl — COMMENTED** [review]
@kerthcet Can you update the verify-generated-yaml script to adjust this modification?

**Comment by kerthcet** [comment]
I followed the advice to update the yaml
```
Makefile:153: recipe for target 'verify-generated-yaml' failed
The Generated yaml is different from the one in installer/volcano-development.yaml
please run 'make generate-yaml TAG=latest RELEASE_DIR=installer 		&& mv /home/runner/work/volcano/volcano/src/github.com/volcano-sh/volcano/hack/../installer/volcano-latest.yaml /home/runner/work/volcano/volcano/src/github.com/volcano-sh/volcano/hack/../installer/volcano-development.yaml' to update
```

But seems not right? I'm clearing the `installer/volcano-monitoring-latest.yaml`.

**Comment by hwdef** [comment]
please try `make update-development-yaml`

File: config/crd/bases/batch.volcano.sh_jobs.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.6.0
+    controller-gen.kubebuilder.io/version: v0.6.2
```
Comment (hwdef) [inline config/crd/bases/batch.volcano.sh_jobs.yaml:7]: This should stay as it is

**Comment by kerthcet** [comment]
> I followed the advice to update the yaml
> 
> ```
> Makefile:153: recipe for target 'verify-generated-yaml' failed
> The Generated yaml is different from the one in installer/volcano-development.yaml
> please run 'make generate-yaml TAG=latest RELEASE_DIR=installer 		&& mv /home/runner/work/volcano/volcano/src/github.com/volcano-sh/volcano/hack/../installer/volcano-latest.yaml /home/runner/work/volcano/volcano/src/github.com/volcano-sh/volcano/hack/../installer/volcano-development.yaml' to update
> ```
> 
> But seems not right? I'm clearing the `installer/volcano-monitoring-latest.yaml`.

Reinstall the helm solves the problem. Ping @hwdef for LGTM.

**Comment by kerthcet** [comment]
The test failure seems  not related with this PR
```
      <*errors.errorString | 0xc000386010>: {
          s: "[Wait time out]: expected job 'pytorch-job' to be in status Completed, actual get Running",
      }
      [Wait time out]: expected job 'pytorch-job' to be in status Completed, actual get Running
```

**Comment by hwdef** [comment]
Thanks for your contribution. 
/lgtm

**Comment by hwdef** [comment]
> /retest ?

I think the `/reset` command does not work. You can close this pr and reopen it to re-trigger the CI run.

**Comment by hwdef** [comment]
By the way, It's better to squash your commits, or improve the information of the second commit.

**Comment by kerthcet** [comment]
> By the way, It's better to squash your commits, or improve the information of the second commit.

It's only for review, then I'll squash the commit.

**Comment by kerthcet** [comment]
Squashed since got LGTMed. Also helps to verify the failure test. cc @hwdef

**Comment by hwdef** [comment]
Thanks! 
/lgtm

**Pull request merged**